services-camel/src/main/java/edu/unc/lib/cdr/routing/MetaServicesRouter.java

```diff
@@ -29,7 +28,9 @@
 	public void configure() throws Exception {
 		from("{{fcrepo.stream}}")
 		.routeId("MetaServicesRouter")
-		.routingSlip(new SimpleExpression("direct-vm:index.start,direct-vm:createThumbnail"));
+		.to("direct-vm:index.start")
+		.multicast()
+		.to("direct-vm:createThumbnail","direct-vm:extractFulltext");
```

Comment:
We don't need to figure it out for this PR, but I would still like to figure out if there's anything we can do about losing the messages midway through.  I may look at that some tomorrow